main.nu

```diff
@@ -39,7 +39,7 @@ def get-ldflags [version: string] {
     const go_import_path = "github.com/restechnica/semverbot"
 
     let ldflags = [
-        $"\"-X ($go_import_path)/internal/ldflags.Version=($version)\""
+        $"-X ($go_import_path)/internal/ldflags.Version=($version)"
```

Comment:
This might have to stay as-is, has this been tested on windows?

Reply:
Haven't tested this on windows, but when I run a `make build` on Mac, I get follow error message:

```sh
make build
building local binary...
invalid value "\"-X github.com/restechnica/semverbot/internal/ldflags.Version=dev\"" for flag -ldflags: parameter may not start with quote character "
```

Reply:
hmm alright, let's see if the pipeline runs - I will fix it if it fails